src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java

```diff
@@ -16,6 +16,8 @@
  
 	private static final String INDEX_PAGE_KEY = "index";
 	private static final String FINAL_PAGE_KEY = "finalPage";
+	private static final String ERROR_PAGE_KEY = "error";
+	private static final String BACK_BUTTON_KEY = "back";
 	private final String MESSAGE_MAP_KEY = "input_message";
 	private final static Logger logger = LoggerFactory.getLogger(ApplicationController.class);
```

Comment:
`private static final` is preferred, for `private static final` constant, the constant name should be all capitalized